### PR TITLE
Upstream unprivileged mode

### PR DESCRIPTION
**Pull request by cathy-zhou**
**- What this PR does and why is it needed**

**- Special notes for reviewers**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: go-controller/pkg/cni/cni.go [hunk 1]
```diff
@@ -132,6 +132,7 @@ func (pr *PodRequest) cmdAdd(kubeAuth *KubeAPIAuth, podLister corev1listers.PodL
 			return nil, err
 		}
 	} else {
+		podInterfaceInfo.PodUID = pr.PodUID
```
Comment (dcbw) [inline go-controller/pkg/cni/cni.go:135]: @cathy-zhou might as well do this for both cases?

File: go-controller/pkg/cni/cnishim.go [hunk 2]
```diff
@@ -209,11 +209,18 @@ func (p *Plugin) CmdAdd(args *skel.CmdArgs) error {
 			klog.Error(err.Error())
 			return err
 		}
+
+		podInterfaceInfo := response.PodIFInfo
+		if err := pr.checkOrUpdatePodUID(podInterfaceInfo.PodUID); err != nil {
```
Comment (dcbw) [inline go-controller/pkg/cni/cnishim.go:214]: @cathy-zhou is this necessary now that cni.go fills in the pod UID for the unprivileged case?

File: go-controller/pkg/cni/cni.go (hunk 1 above)
Comment (cathy-zhou) [reply go-controller/pkg/cni/cni.go]: @dcbw I've make changes according to your commets and repushed, can you please take a look?

**Comment by dcbw** [comment]
LGTM thanks!

**Pull request merged**